tools/lkl/cptofs.c

```diff
@@ -613,7 +613,9 @@ int main(int argc, char **argv)
 			break;
 	}
 
-	ret = lkl_umount_dev(disk_id, cla.part, 0, 1000);
+	umount_ret = lkl_umount_dev(disk_id, cla.part, 0, 1000);
+	if (umount_ret)
```

Comment:
Perhaps we should do it the other way around: 

```
if (ret == 0)
    ret = umount_ret;
```

to preserve the copy error even when umount fails?

Reply:
Yes, sure. I've updated the commit.